config/profiler/run_profiler_integration_tests.sh

```diff
@@ -74,7 +70,6 @@ then
   aws s3 cp s3://smdebug-testing/datasets/cifar-10-python.tar.gz data/cifar-10-batches-py.tar.gz >/dev/null 2>/dev/null # mask output
   aws s3 cp s3://smdebug-testing/datasets/MNIST_pytorch.tar.gz data/MNIST_pytorch.tar.gz >/dev/null 2>/dev/null # mask output
   cd $CODEBUILD_SRC_DIR_TESTS/tests/scripts/pytorch_scripts/data
-  cd data
```

Comment:
Why?

Reply:
The line before actually cd's directly into the `data` directory, so this line simply errors. Doesn't affect the behavior of the script but makes the logs more clean.